DetectorDescription/Core/src/DDCompactView.cc

```diff
@@ -34,23 +34,17 @@ class DDDivision;
    \todo define a stable interface for navigation (don't expose the user to the graph!)
 */    
 // 
-DDCompactView::DDCompactView(const DDLogicalPart & rootnodedata)
+DDCompactView::DDCompactView( const DDLogicalPart & rootnodedata )
   : rep_( new DDCompactViewImpl( rootnodedata )),
```

Comment:
better to use `make_unique`

Reply:
The make_unique function is equivalent to: 
```
unique_ptr<T>(new T(std::forward<Args>(args)...))
```
This is a case where using make_unique will be more verbose then new. rep_ is a unique_ptr

Reply:
But it avoids the explicit use of `new`, which is preferable.